src/goto-instrument/contracts/dynamic-frames/dfcc_utils.h

```diff
@@ -35,6 +35,15 @@ class dfcc_utilst
   namespacet ns;
 
 public:
+  /// \brief Genrate a body `assert(false); assume(false);`
+  /// for the given \p function_id.
+  /// \param function_id function to generate the body for
+  /// \param source_location source location to use for the assertion.
+  /// Must define the property class, comment for the assertion, etc.
+  void gen_fatal_assertion_body(
```

Comment:
Why the abbreviation "gen" vs "generate", which we are using elsewhere?

Reply:
removed